[docs] External Links Errors - saml.xml.org http -> https redirect

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

docs

### Describe the bug

Noticed a new External Link failure while making another PR:

```
[ERROR] Failures: 
[ERROR]   ExternalLinksTest.checkExternalLinks:26 Broken links (1):
                - http://saml.xml.org/saml-specifications (invalid redirect to https://saml.xml.org/saml-specifications)
[ERROR]   ExternalLinksTest.checkExternalLinks:26 Broken links (1):
                - http://saml.xml.org/saml-specifications (invalid redirect to https://saml.xml.org/saml-specifications)
```

### Version

main

### Expected behavior

tests pass

### Actual behavior

tests fail

### How to Reproduce?

run ExternalLinksTest

### Anything else?

_No response_

--- a/docs/documentation/securing_apps/topics/overview/supported-protocols.adoc
+++ b/docs/documentation/securing_apps/topics/overview/supported-protocols.adoc
@@ -26,7 +26,7 @@ the request.
 
 ==== SAML 2.0
 
-link:http://saml.xml.org/saml-specifications[SAML 2.0] is a similar specification to OIDC but a lot older and more mature.  It has its roots in SOAP and the plethora
+link:https://saml.xml.org/saml-specifications[SAML 2.0] is a similar specification to OIDC but a lot older and more mature.  It has its roots in SOAP and the plethora
 of WS-* specifications so it tends to be a bit more verbose than OIDC.  SAML 2.0 is primarily an authentication protocol
 that works by exchanging XML documents between the authentication server and the application.  XML signatures and encryption are used to verify requests and responses.
 

--- a/docs/documentation/server_admin/topics/overview/how.adoc
+++ b/docs/documentation/server_admin/topics/overview/how.adoc
@@ -3,7 +3,7 @@
 
 {project_name} is a separate server that you manage on your network.  Applications are configured to point to and
 be secured by this server.  {project_name} uses open protocol standards like link:https://openid.net/developers/how-connect-works/[OpenID Connect]
-or link:http://saml.xml.org/saml-specifications[SAML 2.0] to secure
+or link:https://saml.xml.org/saml-specifications[SAML 2.0] to secure
 your applications.  Browser applications redirect a user's browser from the application to the {project_name} authentication
 server where they enter their credentials.  This redirection is important because users are completely isolated from applications and
 applications never see a user's credentials.  Applications instead are given an identity token or assertion that is cryptographically

--- a/docs/documentation/server_admin/topics/sso-protocols/con-saml.adoc
+++ b/docs/documentation/server_admin/topics/sso-protocols/con-saml.adoc
@@ -4,7 +4,7 @@
 
 === SAML
 [role="_abstract"]
-link:http://saml.xml.org/saml-specifications[SAML 2.0] is a similar specification to OIDC but more mature.  It is descended from SOAP and web service messaging specifications so is generally more verbose than OIDC.  SAML 2.0 is an authentication protocol that exchanges XML documents between authentication servers and applications.  XML signatures and encryption are used to verify requests and responses.
+link:https://saml.xml.org/saml-specifications[SAML 2.0] is a similar specification to OIDC but more mature.  It is descended from SOAP and web service messaging specifications so is generally more verbose than OIDC.  SAML 2.0 is an authentication protocol that exchanges XML documents between authentication servers and applications.  XML signatures and encryption are used to verify requests and responses.
 
 In general, SAML implements two use cases. 
 

